source/content/git-faq.md

```diff
@@ -166,6 +166,9 @@ If a merge conflict is preventing you from merging a Multidev environment, follo
    git push origin master
    ```
 
```

Comment:
Not sure why this is nested here, as it's not related to a merge conflict as far as I can tell. Not sure where else it should go, maybe under our [Multidev FAQ](https://pantheon.io/docs/multidev-faq)

Reply:
I see what you're saying - I put it here because it relates to Multidev issues like the headings above it and came from the same internal, but it doesn't actually fit in this doc at all
I'll find a better place for it

Reply:
Resolved by #5371